.changeset/empty-pens-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend': patch
```

Comment:
Arguably a breaking change since somebody theoretically could be depending on the `authorization` property - looking for feedback on whether this is worth a minor bump.

Reply:
Yeah, it's pretty small, but the cost of marking this as minor is also pretty small.  Maybe worth marking as `minor` here, but with a note in the changeset that indicates how small the break is:

> The `authorization` property is no longer returned on search results when queried. Note: this will only result in a breaking change if you have custom code in your frontend that relies on the `authorization.resourceRef` property on a document.

Reply:
Yeah, you're right! Done in a7226fcde7.